### PR TITLE
Change UISwitches colour to blue in Settings 💅🏻

### PR DESCRIPTION
**Pull request by jdisho**
In this PR, I change the UISwitches colour in Settigns to blue. I think it makes more sense than green since the labelings of GitHawk are blue-ish. Additionally, I refactored some code in that group 😄

### CODE REVIEW

**Comment by rnystrom** [comment]
@jdisho can you share a screenshot?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jdisho** [comment]
simulator screen shot - iphone xs max - 2018-12-01 at 09 17 24
simulator screen shot - iphone xs max - 2018-12-01 at 09 17 32

File: Classes/Settings/DefaultReactionDetailController.swift [hunk 1]
```diff
@@ -22,18 +22,48 @@ class DefaultReactionDetailController: UITableViewController {
     @IBOutlet var heartCell: UITableViewCell!
     @IBOutlet var enabledSwitch: UISwitch!
 
-    weak var delegate: DefaultReactionDelegate?
+    weak var delegate: DefaultReactionDelegate? {
+        didSet {
+            tableView.reloadData()
```
Comment (rnystrom) [inline Classes/Settings/DefaultReactionDetailController.swift]: Why is this necessary?

File: Classes/Settings/SettingsViewController.swift [hunk 2]
```diff
@@ -109,39 +110,38 @@ GitHubSessionListener {
...
+        default: return
         }
     }
 
     // MARK: Private API
 
-    func updateDefaultReaction() {
+    private func configureSwitches(withColor color: UIColor) {
+        [badgeSwitch, pushSwitch, signatureSwitch, markReadSwitch].forEach {
+            $0?.onTintColor = color
```
Comment (rnystrom) [inline Classes/Settings/SettingsViewController.swift]: Could we just update this to use `UIAppearance` instead? Like these:

https://github.com/GitHawkApp/GitHawk/blob/master/Classes/Views/Styles.swift#L124-L136

File: Classes/Settings/SettingsViewController.swift [hunk 3]
```diff
@@ -258,41 +258,41 @@ GitHubSessionListener {
         }
     }
 
-    @IBAction func onBadgeChanged() {
+    private func updateActiveAccount() {
+        accountsCell.detailTextLabel?.text = sessionManager.focusedUserSession?.username ?? Constants.Strings.unknown
+    }
```
Comment (rnystrom) [inline Classes/Settings/SettingsViewController.swift]: why move?

File: Classes/Settings/SettingsViewController.swift [hunk 4]
```diff
@@ -109,39 +110,38 @@ GitHubSessionListener {
...
+        case reviewAccessCell: onReviewAccess()
+        case accountsCell: onAccounts()
+        case githubStatusCell: onGitHubStatus()
+        case reviewOnAppStoreCell: onReviewOnAppStore()
+        case reportBugCell: onReportBug()
+        case viewSourceCell: onViewSource()
+        case setDefaultReaction: onSetDefaultReaction()
+        case signOutCell: onSignOut()
+        case tryTestFlightBetaCell: onTryTestFlightBeta()
+        default: return
```
Comment (rnystrom) [inline Classes/Settings/SettingsViewController.swift]: Nice, I dig this. Can the default just `break` though? In case someone adds beneath it, might be a pain to debug.

File: Classes/Settings/DefaultReactionDetailController.swift [hunk 5]
```diff
@@ -110,12 +104,11 @@ class DefaultReactionDetailController: UITableViewController {
...
     private func updateSections() {
-        tableView.performBatchUpdates({
-            if enabledSwitch.isOn {
-                self.tableView.insertSections(IndexSet(integer: 1), with: .top)
-            } else {
+        tableView.performBatchUpdates({ [weak self] in
+            guard let `self` = self else { return }
+            enabledSwitch.isOn ?
+                self.tableView.insertSections(IndexSet(integer: 1), with: .top) :
                 self.tableView.deleteSections(IndexSet(integer: 1), with: .top)
```
Comment (rnystrom) [inline Classes/Settings/DefaultReactionDetailController.swift]: Can we leave this as an `if`? This is much harder to read

File: Classes/Settings/DefaultReactionDetailController.swift [hunk 6]
```diff
@@ -110,12 +104,11 @@ class DefaultReactionDetailController: UITableViewController {
     }
 
     private func updateSections() {
-        tableView.performBatchUpdates({
-            if enabledSwitch.isOn {
-                self.tableView.insertSections(IndexSet(integer: 1), with: .top)
-            } else {
+        tableView.performBatchUpdates({ [weak self] in
+            guard let `self` = self else { return }
```
Comment (rnystrom) [inline Classes/Settings/DefaultReactionDetailController.swift]: There's no retain cycle w/ `performBatchUpdates`, no need for `weak`

Also if there were, can avoid the `guard` with

```swift
self?.tableView.insertSections(...)
```

Much cleaner

(but should just remove the `weak` here)

**Comment by jdisho** [comment]
I applied the changes as it was discussed @rnystrom 😊
